src/content/blog/2024_Q3.md

```diff
@@ -0,0 +1,99 @@
+
+2) Last July we held our first hybrid ALF Workshop. The week-long, hands-on event was a success with both in person and remote participants. Some of the contributions can be found in our YouTube channel and more details can be found at: https://gitpages.physik.uni-wuerzburg.de/ALF/ALF_Webpage/news/2024-08-17-alf-workshop-2024/ 
+
+## Microgrant Updates
+
+Earlier this year, we celebrated [awarding over 100 microgrants](https://unitary.fund/posts/2024_microgrant_impact/) and this quarter we opened up an application to join the Microgrant Advisory Review Board. If you are interested in applying, please [submit your application](https://airtable.com/appat5MoVPET2opYb/pag4FfvrhnWX0vgle/form) by Friday, October 18. 
+
+**QCoder**, a competitive quantum programming platform, now offers all its content, including problems and editorials, in English. Its second competition took place on 18 August and attracted 183 participants from 20 countries!
+
+top page: https://www.qcoder.jp/
+
+2nd contest page: https://www.qcoder.jp/contests/QPC002
+
+**Toqito** is now a [numfocus affiliated project](https://numfocus.medium.com/august-september-project-updates-e3dac6f86aa8).
+
+
+[**GQuantum Education**](https://physics.knust.edu.gh/mcp/gquantum-education): Teaching an online quantum education course for students in Ghana. Participated in WiSTEM Ghana's Girls Camp 2024, introducing quantum computing to high school senior girls in Ghana. 
+
+
+**Mqt-qudits**: successfully ran quantum circuits generated with 𝗠𝗤𝗧 𝗤𝘂𝗱𝗶𝘁𝘀, the first open-source framework for mixed-dimensional quantum computing, on the state-of-the-art 𝗾𝘂𝗱𝗶𝘁 𝗾𝘂𝗮𝗻𝘁𝘂𝗺 𝗰𝗼𝗺𝗽𝘂𝘁𝗲𝗿 at the University of Innsbruck. [Check out the tool and tutorial here](https://github.com/cda-tum/mqt-qudits).
+
+
+**Q3 Grants**
+- To **Alberto Mercurio, Yi-Te Huang, and Luca Gravina** to further develop **[Documentation for QuantumToolbox.jl](https://github.com/qutip/QuantumToolbox.jl)**, a state-of-the-art Julia package designed for quantum physics simulations.
+- To **Hana KimLee** to further develop **Comprehensive Quantum Tomography Library**, a Julia library that includes standard algorithms such as state/process tomography and randomized benchmarking, as well as more sophisticated machine learning algorithms.
+- To [**Aidan Sims**](https://www.linkedin.com/in/aidan-sims/) to further develop **Python CVXQuad**, a project translating the CVXQuad library to Python and integrating it with the existing Toqito library.
+- To **Maxime Garnier and Thierry Martinez** to further develop **[Graphix](https://github.com/TeamGraphix/graphix)** through a 2025 workshop in Paris to further accelerate development of Graphix.
+
+## Job Opportunities
+- [Member of Technical Staff - Quantum Benchmarking Lead](https://unitary.fund/jobs/mts-benchmarks/)
```

Comment:
Since we list also job posts from other organizations, we should specify the first two are for Unitary Fund.